product-service/handlers/createProduct.js

```diff
@@ -0,0 +1,31 @@
+import { v4 as uuidv4 } from 'uuid';
+import { docClient } from '../awsDocClient.js';
+
+const putItem = async (params) => {
+  console.log('putItem params: ', params);
+  await docClient.put(params).promise();
+};
+
+export const createProduct = async (event) => {
+  const params = {
+    TableName: process.env.ProductsTableName,
+    Item: {
+      'id': uuidv4(),
+      'title': 'newItem'
+    },
+  };
+
+  try {
+    await putItem(params);
+
+    return {
+      statusCode: 201,
+      body: JSON.stringify('Product successfully created!'),
+    };
+  } catch(error) {
+    return {
+      statusCode: 400,
```

Comment:
I suggest that we should check params from event at the beginning and return 400 in case of invalid data. If we have an error from db it could be 500, I think.

Reply:
checked